### DISCUSSION TITLE
Particle Tracing

### DISCUSSION THREAD

**Discussion by hamidsh84** [nextfoam/baram#57] [Ideas]
Thanks a lot for Baram <3
It would be awesome if tracking of dilute particles (something like [this](https://www.afs.enea.it/project/neptunius/docs/fluent/html/th/node239.htm) or [this](https://www.comsol.com/particle-tracing-module)) is added to Baram. Nowadays processing of cells in microfluidic channels is very common in academia and even industry, and for this purpose one usually have to predict cell trajectories (under the influence of various forces, e.g., its own weight).

**Comment by kdg5424** [comment]
Hi @hamidsh84  this is Donggyu.
Thank you for having interest in BARAM.
We have the plan to develop the solver like DPM.
And there are many inquiries about DPM, we will adjust the DPM development priority.

If you have any question or suggestion about BARAM and DPM please never mind asking me.
Thank you again.

**Reply by hamidsh84** [reply]
This is great news! Thank you :)
